tests/main/snapctl-disable-services-install/task.yaml

```diff
@@ -0,0 +1,20 @@
+summary: Check that `snapctl stop --disable` actually stops services on install
+
+prepare: |
+    snap pack $TESTSLIB/snaps/snapctl-disable-services-install
+
+restore: |
+    rm -f snapctl-disable-services-install_1.0_all.snap
+
+execute: |
+    echo "Verify that the install hook finishes while installing"
+    if ! snap install --dangerous snapctl-disable-services-install_1.0_all.snap; then
```

Comment:
Feel free to ignore it. I don't know about others, but I prefer the shell code to be terse, the `if` block can be replaced with this:
```
echo "Verify that the install hook finishes while installing"
snap install --dangerous snapctl-disable-services-install_1.0_all.snap
...
```
this is running with `set -e`, and the command is expected to succeed (otherwise we'd use `! snap install ...`).

Reply:
Agreed, it's cleaner that way.